src/login/components/App/DashboardApp/Settings/Groups/Group.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import i18n from "../../../../../translation/InjectIntl_HOC_factory";
+
+const RenderAdminList = (props) => {
+  let admins = props.group.owners;
+  return (
+    <Fragment>
+      <label>Admin</label>
+      {admins.map((admin, i) => (
+        <p key={admin.identifier}>
+          {i + 1}. {admin.display_name}
+        </p>
+      ))}
+    </Fragment>
+  );
+};
+
+const RenderMemberList = (props) => {
+  let members = props.group.members;
+  return (
+    <Fragment>
+      <label>Member</label>
+      {members.map((member, i) => (
+        <p key={member.identifier}>
+          {i + 1}. {member.display_name}
+        </p>
+      ))}
+    </Fragment>
+  );
+};
+
+const RenderOpenGroup = (props) => {
+  return props.openGroup ? (
+    <div className="open-group">
+      <RenderAdminList {...props} />
+      <RenderMemberList {...props} />
+    </div>
+  ) : null;
+};
+
+const AnimateDropdownIcon = (props) =>
+  props.openGroup ? (
```

Comment:
I suggest that we will change `const AnimateDropdownIcon` into 
<button className={props.openGroup ? "dropdown-open" : "dropdown-closed"}>^</button>
this will clean up and  reduce the code.

I also suggest to move this code inside the `<div className="element-pair"> ` instead of separated as its own function.

Reply:
thanks! :+1: